cbrain_task/civet_macaque/bourreau/civet_macaque.rb

```diff
@@ -537,7 +546,8 @@ def save_results
     # Log information
     self.addlog_to_userfiles_these_created_these([ source_userfile ],[ civetresult ])
     civetresult.move_to_child_of(source_userfile)
-    self.addlog("Saved new CIVET result file #{civetresult.name}.")
+    self.addlog("Saved new #{is_partial ? "partial" : ""} CIVET result file #{civetresult.name}.")
```

Comment:
Add a separate line in the log file to indicate that the result is partial; just the word 'partial' in the original log line is too hard to find.